po/de.po

```diff
@@ -22148,63 +22107,57 @@ msgstr "git submodule--helper name <Pfad>"
 #: builtin/submodule--helper.c:989
 #, c-format
 msgid "* %s %s(blob)->%s(submodule)"
-msgstr ""
+msgstr "* %s %s(blob)->%s(submodule)"
 
 #: builtin/submodule--helper.c:992
 #, c-format
 msgid "* %s %s(submodule)->%s(blob)"
-msgstr ""
+msgstr "* %s %s(submodule)->%s(blob)"
 
 #: builtin/submodule--helper.c:1005
 #, c-format
 msgid "%s"
-msgstr ""
+msgstr "%s"
 
 #: builtin/submodule--helper.c:1055
-#, fuzzy, c-format
+#, c-format
 msgid "couldn't hash object from '%s'"
-msgstr "Konnte Objekt '%s' nicht parsen."
+msgstr "konnte Hash nicht vom Objekt '%s' erzeugen"
```

Comment:
"konnte Objekt von '%s' nicht hashen" oder "konnte Hash von Objekt von '%s' nicht erzeugen"

Reply:
ja, viel besser, danke!

Reply:
"hashen" hätten wir dann das erste Mal so übersetzt, was ich ungern "eindeutschen" wollen würde..
Wäre "Hash des Objektes von '%s' konnte nicht erzeugt werden" auch in Ordnung? Für %s wird ein Submodul-Pfad eingesetzt.

Reply:
Ja, dann würde ich aber "Hash eines Objektes ..." schreiben.

Reply:
Ja, besser, danke! 😃